### PR TITLE
Features/beekeeper management

### PR DESCRIPTION
**Pull request by vinay-ebi**
> New feature added to Stop, Restart, Skip  WorkFlow.
> Launch remote job async and monitor the through new celery task event_job_status
> New config class pysaga added  
> bsub command added  to lauch the initpipeline on codon 
> Added AMQP publisher to report the workflow status using ES.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,19 +57,51 @@ To use a standalone gunicorn server with 4 worker threads:
...
+=====================
+configure and setup ensembl-prodinf-reporting (https://github.com/Ensembl/ensembl-prodinf-reporting) to update the event workflow status in Elastic search.
+
+Add [DEFAULT] section in .config.ini file (https://github.com/Ensembl/ensembl-prodinf-reporting/blob/main/.config.ini).
+
+```
+[DEFAULT]
+DEBUG=True
+REPORTER_TYPE=elasticsearch
+AMQP_HOST=localhost
```
Comment (marcoooo) [inline README.md:71]: I would like to have same parameter set such as we do for handover app settings, so something more like a `uri` would be better. AMQP_HOST="user:pass@host:port".
Even better would be to use the same settings names as Handover, e.g. CELERY_HOST

File: src/ensembl/production/workflow/hive.py [hunk 2]
```diff
@@ -1,12 +1,16 @@
 def construct_pipeline(job, spec):
     "construct cmd for pipeline to run"
-    hive_dbname = spec['user'] + '_' + job['PipelineName'] + '_' + str(spec['ENS_VERSION']) + '_QRP'
+    
+    hive_dbname = spec['user'] + '_' + job['PipelineName'] + '_' + str(spec['ENS_VERSION']) 
+    queue_name = 'production-rh74' if job.get('HOST', None) == 'NOAH' else 'debug'
```
Comment (marcoooo) [inline src/ensembl/production/workflow/hive.py]: No need for that, consider that either everything is run from same host. `OR` we could consider having the target "run / config" host set up in config files instead.

File: src/ensembl/production/workflow/tpl/core/core.json.tpl [hunk 3]
```diff
@@ -45,6 +47,7 @@
 
     {% block RNAGeneXref %}
         {
+            "HOST": "CODON",
```
Comment (marcoooo) [inline src/ensembl/production/workflow/tpl/core/core.json.tpl:50]: Could be a new bloc with running "host" configuration

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vinay-ebi) [reply README.md:71]: AMQP_HOST param is  related to reporting app (.config.ini). event app register to reporting app to publish the message to elasticsearch . This example is to step all depents app required for event app.

**Review by marcoooo — APPROVED** [review]

**Pull request merged**